Build docker image with maven failed

**Describe the bug**
Building docker image as instructions in document failed with output:
```
[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.4.11:build (default-cli) on project rocketmq-exporter: Exception caught: ADD failed: file not found in build context or excluded by .dockerignore: stat rocketmq-exporter-0.0.2-SNAPSHOT-exec.jar: file does not exist -> [Help 1]

```

**To Reproduce**
Steps to reproduce the behavior:
1. Run ` mvn package -Dmaven.test.skip=true docker:build`
2. The building process will end with error.

**Expected behavior**
The docker image should be successfully created.

--- a/pom.xml
+++ b/pom.xml
@@ -126,12 +126,16 @@
                 </dependencies>
                 <configuration>
                     <imageName>${docker.image.prefix}/${project.artifactId}</imageName>
+                    <imageTags>
+                        <imageTag>${project.version}</imageTag>
+                        <imageTag>latest</imageTag>
+                    </imageTags>
                     <dockerDirectory>src/main/docker</dockerDirectory>
                     <resources>
                         <resource>
                             <targetPath>/</targetPath>
                             <directory>${project.build.directory}</directory>
-                            <include>${project.build.finalName}.jar</include>
+                            <include>${project.build.finalName}-exec.jar</include>
                         </resource>
                     </resources>
                 </configuration>

--- a/src/main/docker/Dockerfile
+++ b/src/main/docker/Dockerfile
@@ -17,6 +17,6 @@
 
 FROM java:8
 MAINTAINER breeze
-ADD rocketmq-exporter-0.0.2-SNAPSHOT-exec.jar quickstart.jar
+ADD rocketmq-exporter-0.0.2-SNAPSHOT-exec.jar rocketmq-exporter.jar
 EXPOSE 5557
-ENTRYPOINT ["java","-jar","quickstart.jar"]
+ENTRYPOINT ["java","-jar","rocketmq-exporter.jar"]

